feat(react-scheduler): add Vertical Grouping (#2740)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/reference/week-view.md

Field | Type | Description
------|------|------------
cellsData | Array&lt;Array&lt;[WeekView.CellData](#weekviewcelldata)&gt;&gt; | Specifies the cells meta data.
groups? | Array&lt;Array&lt;[Group](./grouping-panel.md#group)&gt;&gt; | Specifies the final representation of Scheduler's groups and the order they will be rendered in.
groupOrientation? | [GroupOrientation](./grouping-state.md#grouporientation) | Scheduler's grouping orientation: either 'Vertical' or 'Horizontal'.

> height? | number | Specifies TimeScale's height in pixels.

Reviewer: ```suggestion
height? | number | Specifies the height of the time scale in pixels.
```

> showAllDayTitle? | boolean | If `true`, the 'All Day' title will be displayed.

Reviewer: ```suggestion
showAllDayTitle? | boolean | Specifies whether to display the "All Day" title.
```

labelComponent | ComponentType&lt;[WeekView.TimeScaleLabelProps](#weekviewtimescalelabelprops)&gt; | A component that renders a time scale label.

> allDayTitleComponent? | ComponentType&lt;[AllDay.TitleCellProps](./all-day-panel.md#alldaypaneltitlecellprops)&gt; | A component that renders a title cell.

Reviewer: ```suggestion
allDayTitleComponent? | ComponentType&lt;[AllDay.TitleCellProps](./all-day-panel.md#alldaypaneltitlecellprops)&gt; | A component that renders a cell displaying the "All Day" text.
```

Field | Type | Description
------|------|------------
cellsData | Array&lt;Array&lt;[WeekView.CellData](#weekviewcelldata)&gt;&gt; | Specifies the cells meta data.

> allDayCellsData? | Array&lt;Array&lt;[AllDayPanel.CellData](./all-day-panel.md#alldaypanelcelldata)&gt;&gt; | All day cells meta data.

Reviewer: ```suggestion
allDayCellsData? | Array&lt;Array&lt;[AllDayPanel.CellData](./all-day-panel.md#alldaypanelcelldata)&gt;&gt; | Information about cells on the All Day Panel.
```

cellComponent | ComponentType&lt;[WeekView.TimeTableCellProps](#weekviewtimetablecellprops)&gt; | A component that renders a time table cell.
rowComponent | ComponentType&lt;[WeekView.RowProps](#weekviewrowprops)&gt; | A component that renders a time table row.

> allDayCellComponent? | ComponentType&lt;[AllDay.CellProps](./all-day-panel.md#alldaypanelcellprops)&gt; | A component that renders an All Day panel cell.

Reviewer: ```suggestion
allDayCellComponent? | ComponentType&lt;[AllDay.CellProps](./all-day-panel.md#alldaypanelcellprops)&gt; | A component that renders a cell on the All Day Panel.
```

> allDayRowComponent? | ComponentType&lt;[AllDay.RowProps](./all-day-panel.md#alldaypanelrowprops)&gt; | A component that renders an All Day panel row.

Reviewer: ```suggestion
allDayRowComponent? | ComponentType&lt;[AllDay.RowProps](./all-day-panel.md#alldaypanelrowprops)&gt; | A component that renders a row on the All Day Panel.
```

------|------|------------
startDate? | Date | Specifies the cell's start time.
endDate? | Date | Specifies the cell's end time.

> groupOrientation? | [GroupOrientation](./grouping-state.md#grouporientation) | Scheduler's grouping orientation: either 'Vertical' or 'Horizontal'.

Reviewer 2: Very strange see it here)

Author: It's necessary to use it along with `endOfGroup` prop to render borders correctly
